bio_tools/modules/dna_rna_tools_modules.py

```diff
@@ -0,0 +1,263 @@
+DIFF_DNA_RNA = {
+    "T": "U",
+    "t": "u",
+    "U": "T",
+    "u": "t"
+}
+DNA_TO_RNA = {
+    "A": "U",
+    "T": "A",
+    "G": "C",
+    "C": "G",
+    "a": "u",
+    "t": "a",
+    "g": "c",
+    "c": "g",
+}
+RNA_TO_RNA = {
+    "A": "U",
+    "U": "A",
+    "G": "C",
+    "C": "G",
+    "a": "u",
+    "u": "a",
+    "g": "c",
+    "c": "g",
+}
+DNA_TO_DNA = {
+    "A": "T",
+    "T": "A",
+    "G": "C",
+    "C": "G",
+    "a": "t",
+    "t": "a",
+    "g": "c",
+    "c": "g",
+}
+RNA_TO_DNA = {
+    "A": "T",
+    "U": "A",
+    "G": "C",
+    "C": "G",
+    "a": "t",
+    "u": "a",
+    "g": "c",
+    "c": "g",
+}
+
+
+# transcribe — вернуть транскрибированную последовательность
+def transcribe(arg) -> str:
+    """
+    Transcribes the given DNA or RNA sequence.
+
+    This function replaces nucleotides in the sequence based on transcription
+    rules. For example, DNA adenine (A) is transcribed to RNA uracil (U), and
+    vice versa.
+
+    Parameters
+    ----------
+    arg : str
+        A nucleotide sequence: either DNA or RNA.
+
+    Returns
+    -------
+    result : str
+        The transcribed RNA or DNA nucleotide sequence.
+    """
+    result = ""
+
+    for letter in arg:
+        if letter in DIFF_DNA_RNA:
+            result += DIFF_DNA_RNA[letter]
+        else:
+            result += letter
+    return result
```

Comment:
здесь можно в целом вместо += использовать списки и join 
```suggestion
result = [DIFF_DNA_RNA.get(letter, letter) for letter in arg]
return "".join(result)
```